packages/realm/src/Object.ts

```diff
@@ -505,12 +505,12 @@ export class RealmObject<T = DefaultObject, RequiredProperties extends keyof Omi
    * @note Adding the listener is an asynchronous operation, so the callback is invoked the first time to notify the caller when the listener has been added.
    * Thus, when the callback is invoked the first time it will contain empty array for `changes.changedProperties`.
    */
-  addListener(callback: ObjectChangeCallback<T>): void {
+  addListener(callback: ObjectChangeCallback<T>, keyPaths?: string[]): void {
```

Comment:
Shouldn't this be `keyPaths?: string | string[]`? If so, also check other potential updates such as `ObjectListeners.ts`. Could also add some tests using the `string` alternative (in addition to the currently used `string[]` tests).
```suggestion
  addListener(callback: ObjectChangeCallback<T>, keyPaths?: string | string[]): void {
```

Reply:
Oh yeah 🤦 I forgot all about this agian.